test/e2e_flags.go

```diff
@@ -65,12 +65,13 @@ type EventingEnvironmentFlags struct {
 	Channels
 }
 
-// initializeEventingFlags registers flags used by e2e tests, calling flag.Parse() here would fail in
+// InitializeEventingFlags registers flags used by e2e tests, calling flag.Parse() here would fail in
 // go1.13+, see https://github.com/knative/test-infra/issues/1329 for details
-func initializeEventingFlags() *EventingEnvironmentFlags {
+func InitializeEventingFlags() {
 	f := EventingEnvironmentFlags{}
 
 	flag.Var(&f.Channels, "channels", "The names of the channels, which are separated by comma.")
+	flag.Parse()
```

Comment:
Hmmm... You're removing the 1.13 fix, I can't help but think that the initialization logic here can be done better.

Reply:
Also what Chao told me.

Reply:
I tried it with Go 1.13 and it works.